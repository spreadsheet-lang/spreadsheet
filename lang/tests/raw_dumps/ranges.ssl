A3:A5 = 1
