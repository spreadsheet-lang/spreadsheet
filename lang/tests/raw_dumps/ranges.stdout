ROOT@0..10
  STATEMENT@0..10
    ASSIGN@0..9
      PLACE@0..5
        CELL_RANGE@0..5
          CELL@0..2 "A3"
          COLON@2..3 ":"
          CELL@3..5 "A5"
      WHITESPACE@5..6 " "
      EQ@6..7 "="
      EXPR@7..9
        WHITESPACE@7..8 " "
        INT@8..9 "1"
    NEWLINE@9..10 "\n"
